### PR TITLE
Upgrade Keeper to v0.12 and apply new implementations of OEP-8 and OEP-9

### PR DESCRIPTION
**Pull request by Pedro-vk**
## Description

Upgrade Keeper to v0.12 and apply new implementations of OEP-8 and OEP-9

## Is this PR related with an open issue?

Closes #288

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,21 +19,28 @@ before_script:
...
-  - export EVENTS_HANDLER_VERSION=v0.1.2
+  - export AQUARIUS_VERSION=v0.4.2
+  - export BRIZO_VERSION=v0.5.2
+  - export KEEPER_VERSION=v0.12.0
+  - export EVENTS_HANDLER_VERSION=v0.2.0
   - export KEEPER_OWNER_ROLE_ADDRESS="0xe2DD09d719Da89e5a3D0F2549c7E24566e947260"
+  - rm -rf "${HOME}/.ocean/keeper-contracts/artifacts"
   - bash -x start_ocean.sh --no-pleuston --local-spree-node 2>&1 > start_ocean.log &
   - cd ..
+  - |
```
Comment (kremalicious) [inline .travis.yml]: isn't this handled in `scripts/keeper.sh`?

File: test/ddo/DDO.test.ts [hunk 2]
```diff
@@ -77,71 +75,45 @@ describe('DDO', () => {
...
-                        inLanguage: 'en',
-                        categories: ['Economy', 'Data Science'],
-                        tags: [
-                            'weather',
-                            'uk',
-                            '2011',
-                            'temperature',
-                            'humidity'
-                        ],
                         price: 10,
```
Comment (kremalicious) [inline test/ddo/DDO.test.ts]: `price` should be a string here

File: test/testdata/ddo.json [hunk 3]
```diff
@@ -152,39 +144,47 @@
...
-          "description": "Weather information of UK including temperature and humidity",
           "dateCreated": "2012-10-10T17:00:000Z",
           "author": "Met Office",
           "license": "CC-BY",
-          "copyrightHolder": "Met Office",
-          "workExample": "423432fsd,51.509865,-0.118092,2011-01-01T10:55:11+00:00,7.2,68",
-          "contentUrls": [
-            "https://testocnfiles.blob.core.windows.net/testfiles/testzkp.zip"
-          ],
+          "price": 10,
```
Comment (kremalicious) [inline test/testdata/ddo.json]: here too, should be a string

File: .travis.yml (hunk 1 above)
Comment (Pedro-vk) [reply .travis.yml]: I didn't see this change, was a test from Enrique or Aitor, I'll try to remove it.

**Comment by kremalicious** [comment]
Is this making #288 obsolete?

**Comment by Pedro-vk** [comment]
> Is this making #288 obsolete?

Yes, I think that I used these branch

**Comment by Pedro-vk** [comment]
The PR says that has conflicts but is not true. I think that is produced because of changing the base branch, so I will merge it manually.

**Pull request merged**